README.md

```diff
@@ -1,4 +1,5 @@
-## IBM MQ samples and patterns
+## IBM MQ samples and patterns 
+Please Note: the instructions below should NOT be considered for ibm-messagin-mq-cloud-showcase-app
```

Comment:
This readme needs an overall. The Note: is above a statement that says "you don't want to spend countless hours". I think that still applies to the showcase.